learning_curve.py

```diff
@@ -1,17 +1,19 @@
-import numpy as np
-from matplotlib import pyplot as plt
+"""
+Learning curve
+"""
+from test import Transformer
+from test.loss import OZELoss
```

Comment:
Import order should be:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

See the [PEP8](https://www.python.org/dev/peps/pep-0008/#imports).

Reply:
Got it!